celavi/pylca_celavi/pylca_opt_foreground.py

```diff
@@ -7,6 +7,20 @@
 """
 
 
+# TODO: Consider generalizing "pylca_opt_foreground.py",
```

Comment:
I removed docstring TODOs and left the rest as above

Reply:
Done in PR # 101